### PR TITLE
Update Lab Number Constraint

### LINKED ISSUE #98: [PE-D] labadd l/0 not allowed
labadd l/0 not allowed even though it is quite common for mods to have a Lab 0

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: jetrz/ped#1

### PR DESCRIPTION
**Pull request by bryans17**
Fixes #98, #99.

### CODE REVIEW

File: src/main/java/seedu/address/model/lab/Lab.java [hunk 1]
```diff
@@ -15,12 +15,12 @@
...
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Lab number should be a valid positive integer";
+            "Lab number should be an integer between 0 and 20 inclusive.";
 
     /*
      * Lab number has to be a positive Integer.
      */
-    public static final String VALIDATION_REGEX = "[1-9]\\d*";
+    public static final String VALIDATION_REGEX = "[0-9]\\d*";
```
Comment (channne) [inline src/main/java/seedu/address/model/lab/Lab.java:23]: Hmm I think this can be simplified to `\\d*` (but same effect)

**Review by channne — APPROVED** [review]
Thanks for fixing this 😄  Just a very small suggestion that is more cosmetic than anything haha

File: src/test/java/seedu/address/model/lab/LabTest.java [hunk 2]
```diff
@@ -40,13 +34,16 @@ public void isValidLab() {
         assertFalse(Lab.isValidLab(" ")); // spaces only
 
         // invalid lab
-        assertFalse(Lab.isValidLab("1a")); // invalid lab number
-        assertFalse(Lab.isValidLab("01")); // invalid lab number
+        assertFalse(Lab.isValidLab("1a")); // invalid lab number, includes characters
+        assertFalse(Lab.isValidLab("123456789")); // larger than 20
+        assertFalse(Lab.isValidLab("21")); // larger than 20
+        assertFalse(Lab.isValidLab("2147483648")); // larger than max int value
```
Comment (geetengtan) [inline src/test/java/seedu/address/model/lab/LabTest.java:40]: maybe can add negative int as TC also?

**Review by geetengtan — COMMENTED** [review]
LGTM 👍 Some minor suggestions

File: src/main/java/seedu/address/model/lab/Lab.java (hunk 1 above)
Comment (bryans17) [reply src/main/java/seedu/address/model/lab/Lab.java:23]: Oh yes thanks for the suggestions

File: src/test/java/seedu/address/model/lab/LabTest.java (hunk 2 above)
Comment (bryans17) [reply src/test/java/seedu/address/model/lab/LabTest.java:40]: Yes i should.

**Pull request merged**